packages/eas-cli/src/credentials/android/utils/googleServiceAccountKey.ts

```diff
@@ -77,3 +89,73 @@ function formatGoogleServiceAccountKey({
   line += chalk.gray(`\n    Updated: ${fromNow(new Date(updatedAt))} ago,`);
   return line;
 }
+
+export async function detectGoogleServiceAccountKeyPathAsync(
+  projectDir: string
+): Promise<string | null> {
+  const foundFilePaths = await glob('**/*.json', {
+    cwd: projectDir,
+    ignore: ['app.json', 'package*.json', 'tsconfig.json', 'node_modules'],
+  });
+
+  const googleServiceFiles = foundFilePaths
+    .map(file => path.join(projectDir, file))
+    .filter(fileIsServiceAccountKey);
+
+  if (googleServiceFiles.length > 1) {
+    const selectedPath = await displayPathChooserAsync(googleServiceFiles, projectDir);
+
+    if (selectedPath !== false) {
+      return selectedPath;
+    }
+  } else if (googleServiceFiles.length === 1) {
+    const [detectedPath] = googleServiceFiles;
+
+    if (await confirmDetectedPathAsync(detectedPath)) {
+      return detectedPath;
+    }
+  }
+
+  return null;
+}
+
+async function displayPathChooserAsync(
+  paths: string[],
+  projectDir: string
+): Promise<string | false> {
```

Comment:
Hm, I think this function should return `null` if the user doesn't want to choose any of the suggested files.

Reply:
Otherwise, I think we should rename the function or add a comment explaining what `false` means.